### PR TITLE
Added 15 new snippets and made folders for some snippets for convenience.

### PR DESCRIPTION
**Pull request by skadimoolam**

### CODE REVIEW

**Comment by ybakos** [comment]
@skadimoolam You still didn't make the changes I pointed out from your original pr:
https://github.com/b-g/processing-sublime/pull/93

All of those issues are still in this PR.

**Comment by skadimoolam** [comment]
@ybakos I have made some changes, please review the changes and let me know.

File: Snippets/function/fString.sublime-snippet [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<snippet>
+    <content><![CDATA[String ${1:function_name} (${2:param}) {
```
Comment (ybakos) [inline Snippets/function/fString.sublime-snippet]: There shouldn't be a space between the function name and the parameter list.

File: Snippets/function/fbase.sublime-snippet [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<snippet>
+    <content><![CDATA[${1:float} ${2:function_name} (${3:param}) {
```
Comment (ybakos) [inline Snippets/function/fbase.sublime-snippet]: There shouldn't be a space between the function name and the parameter list.

File: Snippets/function/fbool.sublime-snippet [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<snippet>
+    <content><![CDATA[boolean ${1:function_name} (${2:param}) {
```
Comment (ybakos) [inline Snippets/function/fbool.sublime-snippet]: There shouldn't be a space between the function name and the parameter list.

File: Snippets/function/ffloat.sublime-snippet [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<snippet>
+    <content><![CDATA[float ${1:function_name} (${2:param}) {
```
Comment (ybakos) [inline Snippets/function/ffloat.sublime-snippet]: There shouldn't be a space between the function name and the parameter list.

File: Snippets/function/fint.sublime-snippet [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<snippet>
+    <content><![CDATA[int ${1:function_name} (${2:param}) {
```
Comment (ybakos) [inline Snippets/function/fint.sublime-snippet]: There shouldn't be a space between the function name and the parameter list.

File: Snippets/function/fvoid.sublime-snippet [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<snippet>
+    <content><![CDATA[void ${1:function_name} (${2:param}) {
```
Comment (ybakos) [inline Snippets/function/fvoid.sublime-snippet]: There shouldn't be a space between the function name and the parameter list.

**Comment by ybakos** [comment]
@skadimoolam , I made some more comments.

Let me give you a suggestion. This is a rather large pull request that does multiple things. I strongly recommend creating separate pull requests. For example, one PR for the organization. One PR for additional PVector snippets. One PR for function templates. Etc.

I'm not trying to be difficult, really! Creating separate PRs will help us pick which changes we should keep and which ones we shouldn't.

@b-g I am not a fan of organizing the snippets into subdirectories. This means that we have to "hunt" for snippets in this folder or that folder, rather than seeing them all in one list. (If they are organized into folders, then I think it makes more sense to group them the same way the Processing Reference groups them.)

**Pull request closed without merging**